test_cases/solidity/private_data_on_chain/odd_even/odd_even.sol

```diff
@@ -0,0 +1,32 @@
+/*
+ * @source: https://gist.github.com/manojpramesh/336882804402bee8d6b99bea453caadd#file-odd-even-sol
+ * @author: https://github.com/manojpramesh
+ * Modified by Kaden Zipfel
+ */
+
+pragma solidity ^0.5.0;
+
+contract OddEven {
+    struct Player {
+        address addr;
+        uint number;
+    }
+
+    Player[2] private players;
+    uint8 count = 0;
```

Comment:
No reason to make this a uint8, it will actually use more gas. 

https://youtu.be/qwBkeJ84d2g?t=68

Reply:
Didn't know this, thanks for sharing.